axelor-account/src/main/resources/views/Account.xml

```diff
@@ -461,4 +509,13 @@
     <domain>:id MEMBER OF self.accountSet</domain>
   </action-view>
 
+  <action-view name="action-account-view-reconcile-wizard-form" title=""
+    model="com.axelor.apps.account.db.Account">
+    <view type="form" name="reconcile-wizard-form"/>
+    <view-param name="show-toolbar" value="false"/>
+    <view-param name="show-confirm" value="false"/>
+    <view-param name="popup-save" value="false"/>
+    <view-param name="popup" value="true"/>
```

Comment:
If you want to get the current Account in the context, you can add:
<context name="_showRecord" expr="eval: id"/>

You won't need the "id" field in the form view

Reply:
Solved

Reply:
Sorry the xml line was hidden ... 
You can use the _showRecord instead, it's better and if you do this, in the controller you can just do: 
`Account account = request.getContext().asType(Account.class);`

Reply:
Solved